src/lib/y2users/password_validator.rb

```diff
@@ -68,6 +69,20 @@ def issues
     # @return [Y2Users::User] user containing the password to validate
     attr_reader :user
 
+    LETTERS = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ".freeze
+    NUMBERS = "0123456789".freeze
+    # TODO: find source for this info beside bsc#175706
+    PASSWORD_REGEXP = /\A[-#{NUMBERS}#{LETTERS}!@#\:!\$%^&*() ,;:._+\/|?{}=\['\"`~<>\]\\]+\z/.freeze
```

Comment:
Good point, login.defs has a comment saying why, which we should repeat here:
In non-English locales, A-Z includes different letters than the ASCII range. Notably, in Estonian it excludes T, because Z is a "foreign letter" shoved right after S in the alphabet.

Reply:
due to locales. As e.g. in czech a-z include also ščřčžřáýžááíý characters